language-design.md

```diff
@@ -17,7 +17,41 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+Quote 1: "Why shouldn’t those 'interfaces' be as humanely designed as the ones we tap and swipe?" [Pavlus, 2012].
+
+Here Pavlus asks why programming languages' interfaces are not as "humanely
+designed" as software's graphical interfaces are. This quote stood out to us
+because Pavlus is trying to make a point, but actually ends up asking a
+question that is very easy to answer! One answer is that in general, software
+(programming languages included) is designed towards its users. The software
+Pavlus is referring to is probably targeted at a much broader audience, so it
+needs to be easier. Also, we take issue with Pavlus implying that programming
+is comparable to tapping and swiping. Is Pavlus claiming that programming
+should be as easy as Angry Birds? I doubt that would be a productive language.
```

Comment:
I agree with Adam about the inverse proportion being false, but definitely see your point. Perhaps what you meant to say was that the tools given to a user whilst playing angry birds are minimal and such a small tool set would make for a pretty poor language.